magprime/templates/emails/dept_checklist_intro.txt

```diff
@@ -15,8 +15,8 @@ The Checklist also includes signage and office supplies that you might need from
 
 The following deadlines are part of your Department Head Checklist.
 
-- Creating Shifts Deadline: 11:59pm EST on Tuesday, Oct. 25th
-- Volunteers Assigned to Your Department Deadline: 11:59pm EST on Tuesday, Nov. 1st 
+- Creating Shifts Deadline: 11:59pm EST on Tuesday, Nov. 1st
```

Comment:
Unrelated correction. Ideally these dates wouldn't be hardcoded, but this email has already gone out, so... oh well! We'll get it next year 😄